packages/arb-ts/src/lib/bridge.ts

```diff
@@ -40,6 +38,8 @@ interface BuddyDeployEventResult {
   withdrawalId: BigNumber
   success: boolean
 }
+import { Bridge__factory } from './abi/factories/Bridge__Factory'
```

Comment:
Do we want these on the top of the file?

Reply:
Nice catch 😁